--- /etc/nsswitch.conf.orig	2022-07-05 09:49:46.166847400 +0200
+++ /etc/nsswitch.conf	2022-07-07 15:29:17.989886400 +0200
@@ -11,3 +11,6 @@
 # db_home:  /home/%U
 # db_shell: /bin/bash
 # db_gecos: <empty>
+# Pushed user's home to 'Documents/Cygwin' to have a separate '.ssh' directory
+# for Cygwin's SSH, Windows' SSH, and MobaXterm's SSH
+db_home:  /cygdrive/c/Users/%U/Documents/Cygwin # https://cygwin.com/cygwin-ug-net/ntsec.html#ntsec-mapping-nsswitch
